src/tests/join.test.ts

```diff
@@ -1,53 +1,61 @@
 import * as join from '../join';
 
+const mockGetUserMedia = jest.fn(
+  async () =>
```

Comment:
I'm 99.9% sure this can be
```
() =>
    new Promise<void>(resolve) =>
```

Without the `async` keyword. I've only seen the async keyword needed when using `await` in the function body. I've run into transpilation wackiness in the past when I've done this, but I think nowadays it doesn't happen as often.

Reply:
Thanks, removed :)